SwiftWisdom/Core/UIKit/UIView+GetConstraints.swift

```diff
@@ -3,17 +3,12 @@
 ////  Created by Paul Rolfe on 7/31/15.
 ////  Copyright (c) 2015 Paul Rolfe. All rights reserved.
 ////
```

Comment:
This file was just white space and alignment fixes. The comment removed was a duplicate.